pages/common/fisher.md

```diff
@@ -0,0 +1,32 @@
+# fisher
+
+> Fisher, a fish-shell plugin manager.
+> Install plugins by name or from a managed `fishfile` for bundled installs.
```

Comment:
Change `fishfile` to be in quotes instead of backticks, backticks are usually used when referencing another command.